### PR TITLE
fix object_detection_demo/cpp for yolo models

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
fix double print in obj_detection demo and clamp in yolo wrapper

### CODE REVIEW

File: demos/object_detection_demo/cpp/main.cpp [hunk 1]
```diff
@@ -265,8 +265,6 @@ cv::Mat renderDetectionData(DetectionResult& result, const ColorPalette& palette
         std::ostringstream conf;
         conf << ":" << std::fixed << std::setprecision(1) << obj.confidence * 100 << '%';
         const auto& color = palette[obj.labelID];
-        cv::putText(outputImg, obj.label + conf.str(),
-            cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, { 230, 230, 230 }, 3);
         cv::putText(outputImg, obj.label + conf.str(),
             cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, color);
```
Comment (ivikhrev) [inline demos/object_detection_demo/cpp/main.cpp:269]:
```suggestion
        putHighlightedText(outputImg, obj.label + conf.str(),
            cv::Point2f(obj.x, obj.y - 5), cv::FONT_HERSHEY_COMPLEX_SMALL, 1, color, 2);
```
it's better to use `putHighlightedText` fucntion for labels.

**Review by ivikhrev — CHANGES_REQUESTED** [review]

File: demos/object_detection_demo/cpp/main.cpp (hunk 1 above)
Comment (ivikhrev) [reply demos/object_detection_demo/cpp/main.cpp:269]: btw, it wasn't a double print you fixed, it was the way to highlight text before `putHighlightedText` was added, Just in object detection it wasn't updated for some reason.

Comment (vladimir-dudnik) [reply demos/object_detection_demo/cpp/main.cpp]: ok, committed suggestion

**Review by Wovchena — APPROVED** [review]

**Comment by Wovchena** [comment]
@vladimir-dudnik, can we disable that labler job. It shows PR which didn't run any of our jobs to be green. I suppose it came after https://github.com/openvinotoolkit/open_model_zoo/pull/2741

**Comment by vladimir-dudnik** [comment]
@Wovchena have no objections. I actually not look at labels at all

**Comment by vladimir-dudnik** [comment]
BTW, how this happens? Was not tested at PR related to this model but is tested in other PRs?
18:07:43  models/intel/handwritten-english-recognition-0001/preprocess_gnhk.py: has a shebang line, but isn't executable

**Review by ivikhrev — APPROVED** [review]

**Comment by Wovchena** [comment]
The corresponding PR wasn't tested because it was from external contributor.

**Comment by Wovchena** [comment]
Jenkins please retry a build

**Review by Wovchena — APPROVED** [review]

**Pull request merged**